src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
```

Comment:
You raise a good point but I think that the concern is different. Mine isn't about the spec but the accesibility and UX. What do we win by disabling that feature?